File: app/src/main/java/com/xfl/kakaotalkbot/ResponseParameters.java
package com.xfl.kakaotalkbot;



public class ResponseParameters {
    public String room;
    public String msg;
    public String sender;
    public boolean isGroupChat;
    public SessionCacheReplier replier;
    public ImageDB ImageDB;
    public String packageName;
    public ResponseParameters(String room, String msg, String sender, boolean isGroupChat, SessionCacheReplier replier, ImageDB imageDB,String packName){


        this.room=room;
        this.msg=msg;
        this.sender=sender;
        this.isGroupChat=isGroupChat;
        this.replier=replier;
        this.ImageDB=imageDB;
        this.packageName=packName;
    }
    /*
    @JSGetter
    public String getRoom() {
        return room;
    }

    @JSGetter
    public String getSender(){
        return sender;
    }
    @JSGetter
    public String getMsg(){
        return msg;
    }
    @JSGetter
    public boolean isGroupChat(){
        return isGroupChat;
    }
    @JSGetter
    public SessionCacheReplier replier(){
        return replier;
    }
*/

}

File: app/src/main/java/com/xfl/kakaotalkbot/ResponseParameters.kt
package com.xfl.kakaotalkbot


class ResponseParameters(var room: String, var msg: String, var sender: String, var isGroupChat: Boolean, var replier: SessionCacheReplier, var ImageDB: ImageDB, var packageName: String)/*
    @JSGetter
    public String getRoom() {
        return room;
    }

    @JSGetter
    public String getSender(){
        return sender;
    }
    @JSGetter
    public String getMsg(){
        return msg;
    }
    @JSGetter
    public boolean isGroupChat(){
        return isGroupChat;
    }
    @JSGetter
    public SessionCacheReplier replier(){
        return replier;
    }
*/
